pom.xml

```diff
@@ -489,109 +337,6 @@
-                <version>2.7</version>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <version>3.0.5</version>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.4.1</version>
-                <executions>
-                    <execution>
-                        <id>attach-javadocs</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
         </plugins>
     </build>
-
-    <profiles>
-        <profile>
-            <id>localdev</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.portals.pluto</groupId>
-                        <artifactId>maven-pluto-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <phase>generate-resources</phase>
-                                <goals>
-                                    <goal>assemble</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-war-plugin</artifactId>
-                        <configuration>
-                            <classifier>localdev</classifier>
-                            <warName>${project.artifactId}</warName>
-                            <webXml>${project.build.directory}/pluto-resources/web.xml</webXml>
-                        </configuration>
-                    </plugin>
-                    <plugin>
-                        <artifactId>maven-uportal-plugin</artifactId>
-                        <groupId>org.jasig.portal</groupId>
-                        <configuration>
-                            <contextName>${project.artifactId}</contextName>
-                            <artifactId>${project.artifactId}</artifactId>
-                            <artifactGroupId>${project.groupId}</artifactGroupId>
-                            <artifactVersion>${project.version}</artifactVersion>
-                            <artifactClassifier>localdev</artifactClassifier>
-                            <removeExistingDirectories>true</removeExistingDirectories>
-                            <extractWars>true</extractWars>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
-              <id>doclint-java8-disable</id>
```

Comment:
Since we only support Java 8+ , doclint is disabled at the maven-javadoc-plugin level in the parent pom.